password_security/controllers/main.py

```diff
@@ -44,6 +45,8 @@ def web_login(self, *args, **kw):
             request.params['password']
         )
         if not uid:
+            request.uid = old_uid
+            request.uid = request.env.ref('base.public_user').id
```

Comment:
@nguyenductamlhp - can you remove this line and we can just go with the original solution? I've performed an extensive audit and I think we're good here

Reply:
@lasley, OK